backend/README.md

```diff
@@ -22,7 +22,7 @@ $ docker-compose exec neo4j migrate
 
 {% endtab %}
 
-{% tab title="Without Docker" %}
+{% tab title="Local" %}
```

Comment:
```suggestion
{% tab title="Withouth Docker" %}
```
Yes, I was also tempted to call it that way first. Because we call it "local installation" when we mean "without docker", right? People who don't know docker won't understand what it means so I called it simply "without docker"